docs/index.html

```diff
@@ -19,7 +19,7 @@
       homepage: 'gettingStarted.md',
```

Comment:
The getting started document has incorrect info in it.  Update it with the contents on confluence